Website/js/Debug/dnn.modalpopup.js

```diff
@@ -222,14 +219,21 @@
 
                             var personaBarIFrameWidth = 0;
                             if ($('#personaBar-iframe').length) {
-                                personaBarIFrameWidth = $('#personaBar-iframe').width();
+                                personaBarIFrameWidth = $('#personaBar-iframe').outerWidth();
+                            }
+
+                            var closeButtonWidthCorrection = 0;
+                            var closeButtonHeightCorrection = 0;
+                            if ($('button.ui-dialog-titlebar-close').length) {
+                                closeButtonHeightCorrection = $('button.ui-dialog-titlebar-close').parent('.dnnModalCtrl').height();
+                                closeButtonWidthCorrection = $('button.ui-dialog-titlebar-close').parent('.dnnModalCtrl').width();
                             }
 
-                            newWidth = $window.width() - 40 - personaBarIFrameWidth;
-                            newHeight = $window.height() - 46;
+                            newWidth = $window.outerWidth() - personaBarIFrameWidth - (closeButtonWidthCorrection / 7.5) - 40;
```

Comment:
Hi @Mohtshum , can we add comment to clarify whats the "closeButtonWidthCorrection / 7.5" mean? whats the "- 40" mean? same to below code about "closeButtonWidthCorrection / 5.5" and "closeButtonHeightCorrection / 11", without comment we can't know what that do and will be hard to maintenance.